spec/models/ext_management_system_spec.rb

```diff
@@ -147,13 +147,13 @@
   end
 
   it "validates type" do
-    v = FactoryBot.create(:ems_vmware)
-    e = FactoryBot.create(:ext_management_system)
+    v = FactoryBot.create(:ems_infra)
+    e = FactoryBot.create(:ems_network)
```

Comment:
terribly minor, but the variable name `e` doesn't make sense after this change - maybe `n`?